src/impl/tpls/KokkosBlas1_nrminf_tpl_spec_decl.hpp

```diff
@@ -208,4 +208,166 @@ KOKKOSBLAS1_CNRMINF_TPL_SPEC_DECL_BLAS( Kokkos::LayoutLeft, Kokkos::HostSpace, f
+    } \
+  } \
+};
+
+#define KOKKOSBLAS1_SNRMINF_TPL_SPEC_DECL_CUBLAS( LAYOUT, MEMSPACE, ETI_SPEC_AVAIL ) \
+template<class ExecSpace> \
+struct NrmInf< \
+Kokkos::View<float, LAYOUT, Kokkos::HostSpace, \
+             Kokkos::MemoryTraits<Kokkos::Unmanaged> >, \
+Kokkos::View<const float*, LAYOUT, Kokkos::Device<ExecSpace, MEMSPACE>, \
+             Kokkos::MemoryTraits<Kokkos::Unmanaged> >, \
+1,true, ETI_SPEC_AVAIL > { \
+  \
+  typedef Kokkos::View<float, LAYOUT, Kokkos::HostSpace, \
+                       Kokkos::MemoryTraits<Kokkos::Unmanaged> > RV; \
+  typedef Kokkos::View<const float*, LAYOUT, Kokkos::Device<ExecSpace, MEMSPACE>, \
+                       Kokkos::MemoryTraits<Kokkos::Unmanaged> > XV; \
+  typedef typename XV::size_type size_type; \
+  \
+  static void nrminf (RV& R, const XV& X) \
+  { \
+    const size_type numElems = X.extent(0); \
+    if (numElems == 0) { R() = 0.0f; return; } \
+    if (numElems < static_cast<size_type> (INT_MAX)) { \
+      nrminf_print_specialization<RV,XV>(); \
+      const int N = static_cast<int> (numElems); \
+      constexpr int one = 1; \
+      int idx; \
+      KokkosBlas::Impl::CudaBlasSingleton & s = KokkosBlas::Impl::CudaBlasSingleton::singleton(); \
+      cublasIsamax(s.handle, N, X.data(), one, &idx); \
+      Kokkos::deep_copy(R, subview(X,idx-1)); \
+    } else { \
+      NrmInf<RV,XV,1,false,ETI_SPEC_AVAIL>::nrminf(R,X); \
+    } \
+  } \
+};
+
+#define KOKKOSBLAS1_ZNRMINF_TPL_SPEC_DECL_CUBLAS( LAYOUT, MEMSPACE, ETI_SPEC_AVAIL ) \
+template<class ExecSpace> \
+struct NrmInf< \
+Kokkos::View<double, LAYOUT, Kokkos::HostSpace, \
+             Kokkos::MemoryTraits<Kokkos::Unmanaged> >, \
+Kokkos::View<const Kokkos::complex<double>*, LAYOUT, Kokkos::Device<ExecSpace, MEMSPACE>, \
+             Kokkos::MemoryTraits<Kokkos::Unmanaged> >, \
+1,true, ETI_SPEC_AVAIL > { \
+  \
+  typedef Kokkos::View<double, LAYOUT, Kokkos::HostSpace, \
+                       Kokkos::MemoryTraits<Kokkos::Unmanaged> > RV; \
+  typedef Kokkos::View<const Kokkos::complex<double>*, LAYOUT, Kokkos::Device<ExecSpace, MEMSPACE>, \
+                       Kokkos::MemoryTraits<Kokkos::Unmanaged> > XV; \
+  typedef typename XV::size_type size_type; \
+  typedef Kokkos::Details::InnerProductSpaceTraits<Kokkos::complex<double>> IPT; \
+  \
+  static void nrminf (RV& R, const XV& X) \
+  { \
+    const size_type numElems = X.extent(0); \
+    if (numElems == 0) { R() = 0.0; return; } \
+    if (numElems < static_cast<size_type> (INT_MAX)) { \
+      nrminf_print_specialization<RV,XV>(); \
+      const int N = static_cast<int> (numElems); \
+      constexpr int one = 1; \
+      int idx; \
+      KokkosBlas::Impl::CudaBlasSingleton & s = KokkosBlas::Impl::CudaBlasSingleton::singleton(); \
+      cublasIzamax(s.handle, N, reinterpret_cast<const cuDoubleComplex*>(X.data()), one, &idx); \
+      Kokkos::View<Kokkos::complex<double>, LAYOUT, Kokkos::HostSpace > R_cplx("R_cplx",1); \
```

Comment:
This still calls `new`.  Why not make an unmanaged View for that one entry?
```c++
Kokkos::complex<double> R_cplx_val {0.0, 0.0};
Kokkos::View<Kokkos::complex<double>, LAYOUT, Kokkos::HostSpace> > R_cplx (&R_cplx_val);
```
More importantly, you shouldn't pass 1 in, because this is a rank-0 Kokkos::View.  (Disable deprecated code in Kokkos and see what happens.)  Even more importantly, if this View is managed, then you'll be using `Kokkos::HostSpace::execution_space` to initialize this View, which may not be what you want.  Instead, just use an unmanaged rank-0 View.